school/diary/views.py

```diff
@@ -162,6 +165,26 @@ def param_update(request):
 
 
 def football_player(request):
+    template = loader.get_template("diary/info_football_player.html")
+    context = {
+        "players": FootballPlayer.objects.all()
+    }
+    return HttpResponse(template.render(context, request))
+
+
+def footballer(request, football_player_id):
+    player = get_object_or_404(FootballPlayer, pk=football_player_id)
+    return render(request, "diary/player.html", {"players": player})
+
+
+def achievements(request):
+    ach_template = loader.get_template("diary/achievements_detail.html")
+    context_achievement = {
+        "achievement_key": Achievement.objects.all()
+    }
+    return HttpResponse(ach_template.render(context_achievement, request))
+
 
-    player = FootballPlayer.objects.all()
-    return HttpResponse(player)
+def achievements_detail(request, achievement_id):
+    reaching = Achievement.objects.get(pk=achievement_id)
```

Comment:
it would be better to use `get_object _or_404` or try-except the case if ObjectDoesNotExist